Maquetado/scss/styles.scss

```diff
@@ -3,5 +3,5 @@
 }
 
 .body-container {
-  padding: $nav-height+50px 100px 50px;
+  padding: $nav-height+50px 100px 0 100px;
```

Comment:
you don't need to add the last `100px` if the padding receives 3 values the second one represents left and right